eng/pipelines/corefx-base.yml

```diff
@@ -84,36 +84,28 @@ jobs:
           - group: DotNet-Internal-Tools-Feed
           - group: DotNet-Blob-Feed
 
-        - ${{ if eq(parameters.isOfficialBuild, 'false') }}:
-          - _msbuildCommonParameters: ''
-
         - ${{ if eq(job.submitToHelix, 'true') }}:
           - group: DotNet-HelixApi-Access
           - _archiveTestsParameter: /p:ArchiveTests=Tests
 
+        - _args: -restore -build -configuration $(_BuildConfig) -ci -buildtests -arch $(_architecture) -framework $(_framework) $(_archiveTestsParameter)
+        - _commonArguments: $(_args)
+        - _windowsOfficialBuildArguments: ''
```

Comment:
Can you move this to where you're setting `_msbuildCommonParameters: ''` as well?